Makefile

```diff
@@ -291,12 +303,12 @@ update: update-base update-deps
 # Delete planet4 main theme and plugin
 .PHONY: clean-repos
 clean-repos:
-	rm -fr persistence/app/public/wp-content/themes/planet4-master-theme
+	@rm -fr persistence/app/public/wp-content/themes/planet4-master-theme
```

Comment:
Shouldn't that change also in the line below?

Reply:
I thought so, the [documentation says so](https://www.gnu.org/software/make/manual/html_node/Echoing.html#index-_0040-_0028in-recipes_0029),, but apparently it doesn't; adding a `@` to only the second line doesn't even silence anything, so my guess is that those blocks are considered as a single execution, silencing the first line acts for all